sources/install/package_base.sh

```diff
@@ -199,6 +199,16 @@ function install_gf() {
     add-to-list "gf,https://github.com/tomnomnom/gf,A wrapper around grep to avoid typing common patterns"
 }
 
+function install_cyberchef() {
+    colorecho "Installing CyberChef"
+    mkdir /opt/tools/CyberChef
+    LAST_VERSION=$(wget https://github.com/gchq/CyberChef/releases/latest/download/CyberChef.zip 2>&1 | grep Location: | grep -E -o 'v.*' | cut -d '/' -f 1)
```

Comment:
Could you make the variable `local` so that it does not conflict elsewhere in the code ? By doing so, could you switch the variable name to lowercase ?

Reply:
Maybe the variable content could be tested first before moving on to the other commands